src/block_cache/block_chunkcache.c

```diff
@@ -694,10 +698,23 @@ __wt_chunkcache_reconfig(WT_SESSION_IMPL *session, const char **cfg)
         __wt_spin_lock(session, &chunkcache->hashtable[i].bucket_lock);
         TAILQ_FOREACH_SAFE(chunk, WT_BUCKET_CHUNKS(chunkcache, i), next_chunk, chunk_tmp)
         {
-            if (__chunkcache_should_pin_chunk(session, chunk))
+            if (__chunkcache_should_pin_chunk(session, chunk)) {
+                /*
+                 * Increment the stat only when a chunk, which was initially unpinned, becomes
```

Comment:
NP: I think this would sound better as:
 "Increment the stat (only) when a chunk that was initially unpinned becomes pinned."  
Not sure if 'only' is necessary, but if it requires clarity feel free to keep it there. 
The use of commas are not necessary here.

Reply:
Makes sense, updated in [4e598ca](https://github.com/wiredtiger/wiredtiger/pull/9480/commits/4e598cae83e35364fe35a1a3437acc2428362386). It is in one line now, seems much nicer. :)